signal2html/core.py

```diff
@@ -128,7 +130,38 @@ def add_mms_attachments(db, mms, backup_dir, thread_dir):
         mms.attachments.append(a)
 
 
+def get_mms_mentions(encoded_mentions, addressbook, mid):
+    """Decode mentions encoded in a SQL blob."""
+    mentions = {}
+    if encoded_mentions:
+        try:
+            structured_mentions = StructuredMentions.loads(encoded_mentions)
+        except (ValueError, IndexError, TypeError) as e:
+            logger.warn(
+                f"Failed to load quote mentions for message {mid}: {str(e)}"
+            )
+            return []
+
+        for structured_mention in structured_mentions.mentions:
+            recipient = addressbook.get_recipient_by_uuid(
+                structured_mention.who_uuid
+            )
+            name = recipient.name
+            mention = Mention(
+                mention_id=-1, name=name, length=structured_mention.length
```

Comment:
What is the `mention_id` used for? It doesn't seem to affect anything. Just keeping it for potential future use or so?

Reply:
(No problem, pretty busy myself :) )

I was thinking it could be helpful for debugging at some point. I could still write it out as an id attribute of the generated span (mention-nnn if not -1). However, you're right that beyond that there's not much point, so I could also take it out. What do you think?

Reply:
It's probably fine to keep it then. You could write it out if you prefer, but I don't feel strongly about it either way.

Feel free to merge when you're ready! :)

Reply:
Ok, merging as is :)